parm/config/gfs/config.resources

```diff
@@ -907,10 +907,16 @@ elif [[ ${step} = "eobs" || ${step} = "eomg" ]]; then
     export npe_node_eobs
     export npe_node_eomg=${npe_node_eobs}
     export is_exclusive=True
-    #The number of tasks and cores used must be the same for eobs
-    #For S4, this is accomplished by running 10 tasks/node
+    # The number of tasks and cores used must be the same for eobs
+    # See https://github.com/NOAA-EMC/global-workflow/issues/2092 for details
+    # For S4, this is accomplished by running 10 tasks/node
     if [[ ${machine} = "S4" ]]; then
        export npe_node_eobs=10
+    elif [[ ${machine} = "HERCULES" ]]; then
+       # For Hercules, this is only an issue at C384; use 20 tasks/node
+       if [[ ${CASE} = "C384" ]]; then
+          export npe_node_eobs=20
```

Comment:
This line earlier (line 908):
```
export npe_node_eomg=${npe_node_eobs}
```
...needs to benefit from updating `npe_node_eobs` for S4 and Hercules C384. Didn't notice earlier when you added S4, just seeing this now. :)

Reply:
Good catch, thanks @KateFriedman-NOAA!  Fixed.